bitcoin/src/amount.rs

```diff
@@ -494,6 +494,10 @@ impl Amount {
     pub const ONE_BTC: Amount = Amount(100_000_000);
     /// The maximum value allowed as an amount. Useful for sanity checking.
     pub const MAX_MONEY: Amount = Amount(21_000_000 * 100_000_000);
+    /// The minimum value of an amount.
+    pub const MIN: Amount = Amount::ZERO;
+    /// The maximum value of an amount.
+    pub const MAX: Amount = Amount(u64::MAX);
```

Comment:
I would prefer not to add this until it's resolved whether `Amount` should have `MAX` equal to `MAX_MONEY`.

Reply:
I think we should punt on this for this PR. But my feeling is that no, MAX_MONEY feels too "arbitrary" in a language that already makes unchecked overflows quite difficult.

Reply:
I also note that Bitcoin Core will parse amounts greater than MAX_MONEY, and if we try to use that as an actual type-enforced maximum, then we won't be able to parse the same amounts, which seems like a discrepancy without a lot of value.